src/Displaz.jl

```diff
@@ -326,9 +350,92 @@ function plot3d(plotobj::DisplazWindow, position; color=[1,1,1], markersize=[0.1
     nothing
 end
 
-# Interop with FixedSizeArrays.
-plot3d{V<:FixedVector{3}}(plotobj::DisplazWindow, position::Vector{V}; kwargs...) =
-    plot3d(plotobj, destructure(position); kwargs...)
+
+
+"""
+    mutate!([plotobj,] index; attr1=value1, ...))
+
+Mutate the data in an existing displaz data set, for instance to change the
+position or other attribute of a subset of points (with the advantage of
+reducing the amount of communication between Julia and displaz, and therefore
+increasing speed).
+
+`index` is vector of indices with reference to the original plot. The attribute
+`label` is used to match with the correct data set within displaz. The
```

Comment:
I think the fact that the label depends on the number of vertices this way is just a misfeature from a previous design iteration.  I'd be happy to do away with that.